systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -457,6 +544,20 @@ void testBuildOtherPluginTypeWithAndWithoutFileName() {
     @Tag(ACCEPTANCE)
     @KindNotSupported("using kind we encounter (error building image: deleting file system after stage 0: unlinkat //product_uuid: device or resource busy)")
     @ParallelTest
+    @TestDoc(
+        description = @Desc("Test building a plugin using Maven coordinates artifacts."),
+        steps = {
+            @Step(value = "Create a test storage object", expected = "Test storage object is created"),
+            @Step(value = "Generate image name for the test case", expected = "Image name is generated successfully"),
+            @Step(value = "Create Kafka topic and Kafka Connect resources", expected = "Resources are created and available"),
```

Comment:
I would write here what configuration of KafkaConnect you are providing here (the configuration for plugin using mvn coordinates)

Reply:
Thanks 👍